nss-day-cohort-29/reactive-nutshell-springwynne
# Jb messages with current master merged

# Description

Got messages basic mvp.  Will still need to implement friend add feature though.

## Type of change



# Testing Instructions for Change Made

git fetch and enter branch jb-messagesWithCurrentMasterMerged
open in browser while react and local json 5002 are running

no matter who you login as messages will currently function as though you are user 1.
Since are base json is the same you should be able to log in as Hernando with password yomama
then....
enter Messages section by clicking "messages" navbar.  there you will see messages.  user messages will have an option to edit and will be in a purple-ish box
try an edit by clicking edit...message will then turn to input box where you can change text
then save button will show message with freshly edited text

you can also Send a new message.... type in the input box next to "Send" button
click "Send" and then the new message will be displayed in purple-ish box with ability to edit.

there's some subtle tweaks I still will do but this places feature solidly in master for now and implements basic mvp.


# Checklist:

- [] I have commented my code, particularly in hard-to-understand areas
